packages/react/src/component-tests/IcTooltip/IcTooltipTestData.tsx

```diff
@@ -225,6 +225,26 @@ export const Truncation = () => {
   );
 };
 
+export const LargeLabel = () => {
+  return (
+    <div style={{ margin: "40px" }}>
+      <IcTooltip
+        label="This is a body of text that is purposefully long to test whether text is being wrapped within the tooltip. It has been wrapped based on the neccessity to prevent future bugs and that the user may read the entire body of text. The number of lines in the text is clamped to the number passed through the maxLines prop."
```

Comment:
'neccessity' is a typo, which is causing the build to fail - correct the spelling and we should be good :)

Reply:
necessity has been corrected 👍